openvdb_points_houdini/houdini/SOP_OpenVDB_Points.cc

```diff
@@ -185,112 +195,112 @@ convertAttributeFromHoudini(PointDataTree& tree, const PointIndexTree& indexTree
+                                            TruncateCodec>, false>(tree, indexTree, name, attribute, defaults);
             }
         }
         else if (storage == GA_STORE_REAL64) {
-            convertAttributeFromHoudini<TypedAttributeArray<double> >(tree, indexTree, name, attribute, defaults, compression);
+            convertAttributeFromHoudini<TypedAttributeArray<double>, false>(tree, indexTree, name, attribute, defaults);
         }
     }
     else if (width == 3 || width == 4)
     {
         // note: process 4-component vectors as 3-component vectors for now
 
         if (storage == GA_STORE_INT32) {
-            convertAttributeFromHoudini<TypedAttributeArray<Vec3<int> > >(tree, indexTree, name, attribute, defaults, compression);
+            convertAttributeFromHoudini<TypedAttributeArray<Vec3<int> >, false>(tree, indexTree, name, attribute, defaults);
         }
         else if (storage == GA_STORE_REAL16)
         {
             // implicitly convert 16-bit float into truncated 32-bit float
 
             convertAttributeFromHoudini<TypedAttributeArray<Vec3<float>,
-                                        TruncateCodec> >(tree, indexTree, name, attribute, defaults, compression);
+                                        TruncateCodec>, false>(tree, indexTree, name, attribute, defaults);
         }
         else if (storage == GA_STORE_REAL32)
         {
             if (compression == NONE) {
-                convertAttributeFromHoudini<TypedAttributeArray<Vec3<float> > >(tree, indexTree, name, attribute, defaults, compression);
+                convertAttributeFromHoudini<TypedAttributeArray<Vec3<float> >, false>(tree, indexTree, name, attribute, defaults);
             }
             else if (compression == TRUNCATE) {
                 convertAttributeFromHoudini<TypedAttributeArray<Vec3<float>,
-                                            TruncateCodec> >(tree, indexTree, name, attribute, defaults, compression);
+                                            TruncateCodec>, false>(tree, indexTree, name, attribute, defaults);
             }
             else if (compression == UNIT_VECTOR) {
                 convertAttributeFromHoudini<TypedAttributeArray<Vec3<float>,
-                                            UnitVecCodec> >(tree, indexTree, name, attribute, defaults, compression);
+                                            UnitVecCodec>, false>(tree, indexTree, name, attribute, defaults);
             }
         }
         else if (storage == GA_STORE_REAL64) {
-            convertAttributeFromHoudini<TypedAttributeArray<Vec3<double> > >(tree, indexTree, name, attribute, defaults, compression);
+            convertAttributeFromHoudini<TypedAttributeArray<Vec3<double> >, false>(tree, indexTree, name, attribute, defaults);
         }
     }
     else if (width == 3 && storage == GA_STORE_REAL16) {
-        convertAttributeFromHoudini<TypedAttributeArray<Vec3<half> > >(tree, indexTree, name, attribute, defaults, 1);
```

Comment:
i believe lines 262 to 276 can be removed as they are duplicated of the above branch